include/utility/json_utility.hpp

```diff
@@ -1003,5 +1003,46 @@ inline bool isFruReplaceableAtStandby(const nlohmann::json& i_sysCfgJsonObj,
 
     return false;
 }
+
+/**
+ * @brief API to get list of FRUs replaceable at standby.
```

Comment:
maybe add from JSON also

Reply:
maybe a brief telling that we will return a vector of inventory path of the FRU's that are replaceable at standby.

Reply:
Done.